docs/reference/synonyms/apis/delete-synonyms-set.asciidoc

```diff
@@ -0,0 +1,91 @@
+[[delete-synoyms-set]]
+=== Delete Synonyms Set
+
+beta::[]
+
+++++
+<titleabbrev>Delete Synonyms Set</titleabbrev>
+++++
+
+Deletes a Synonyms Set.
+
+[[delete-synonyms-set-request]]
+==== {api-request-title}
+
+`DELETE _synonyms/<synonyms_set>`
+
+[[delete-synonyms-set-prereqs]]
+==== {api-prereq-title}
+
+* Requires the `manage_search_synonyms` cluster privilege.
+* You can only delete a Synonym Set that is not in use by any <<analysis-synonym-graph-tokenfilter,synonym graph token filters>> and <<analysis-synonym-tokenfilter,synonym token filters>> that are included in an analyzer. See <<delete-synonym-set-analyzer-requirements>> for more information.
```

Comment:
I _think_ we want to lowercase `synonym(s) sets`. Also, I think we want to be consistent with pluralization throughout our documentation and refer to it as a `synonyms set`? (This comment applies throughout the PR)

```suggestion
* You can only delete a synonyms set that is not in use by any <<analysis-synonym-graph-tokenfilter,synonym graph token filters>> and <<analysis-synonym-tokenfilter,synonym token filters>> that are included in an analyzer. See <<delete-synonym-set-analyzer-requirements>> for more information.
```

Reply:
I'm happy to change Synonyms Sets to lowercase. I was taking Search Applications example for this one - I'd like to have guidance from the docs team as I'm not sure of what should be the correct form 👍 

I've fixed the plurals, thanks for catching it!

Reply:
The general guidance is that product names should be Upper Cased and "things" should be lower cased. Search Applications is a weird case because it refers to the branded product name (Search Applications) but at the end of the day it's a search application. 

Now that I've confused you more I'll cede to the docs team 😂

Reply:
My official docs opinion :smile: : plural and lowercase

Reply:
Thank you both! 🙇

Reply:
I'll create a separate PR for doing code changes for being consistent with the plurals - I'd prefer to keep this PR as focused on docs as possible